conf.yaml

```diff
@@ -2473,6 +2448,32 @@ contents:
 
     -   title:      Legacy Documentation
         sections:
+          - title:      Getting Started
+            prefix:     en/elastic-stack-get-started
+            current:    8.2
+            index:      docs/en/getting-started/index.asciidoc
+            branches:   [ {main: master}, 8.3, 8.2, 8.1, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3 ]
```

Comment:
Do we still want to publish main/master and 8.3 versions of this book? Especially if we "freeze" the live versions at 8.2 and 7.17?

Reply:
Nope. Fixed that. :-)